Dockerfile

```diff
@@ -88,6 +88,9 @@ RUN curl -sSL  "https://storage.googleapis.com/golang/go${GO_VERSION}.linux-amd6
 ENV PATH /go/bin:/usr/local/go/bin:$PATH
 ENV GOPATH /go:/go/src/github.com/docker/docker/vendor
 
+# Enable Go vendor experiment to help vendored into packages to bring their own dependencies, if necessary.
+ENV GO15VENDOREXPERIMENT 1
```

Comment:
we can't do both GOPATH magic and GO15VENDOREXPERIMENT... because `vendor/` vs `vendor/src/`